src/usb_hs.rs

```diff
@@ -50,8 +50,8 @@ impl USB1 {
         usb_global: stm32::OTG1_HS_GLOBAL,
         usb_device: stm32::OTG1_HS_DEVICE,
         usb_pwrclk: stm32::OTG1_HS_PWRCLK,
-        _pin_dm: gpio::PA11<Analog>,
-        _pin_dp: gpio::PA12<Analog>,
+        _pin_dm: gpio::PA11<crate::gpio::Analog>,
```

Comment:
This change is because clippy was throwing annoying warnings when the features weren't enabled about an unused `Analog` import